Framework/PythonInterface/mantid/_testing/AssertAlmostEqual.py

```diff
@@ -29,19 +29,24 @@ def assert_almost_equal(Workspace1, Workspace2, rtol=0.0, atol=0.0):
         If atol and rtol are both provided
 
     """
-
-    if rtol != 0.0 and atol != 0.0:
-        raise ValueError("Specify rtol or atol, not both")
+    rtol = kwargs.pop("rtol", None)
+    atol = kwargs.pop("atol", None)
     _rel = False
     tolerance = 1e-10
-    if atol:
-        tolerance = atol
 
-    if rtol:
-        tolerance = rtol
-        _rel = True
-
-    result, message = CompareWorkspaces(Workspace1, Workspace2, Tolerance=tolerance, ToleranceRelErr=_rel)
+    if rtol is not None and atol is not None:
+        raise ValueError("Specify rtol or atol, not both")
+    elif rtol in None and atol is None:
+        raise ValueError("Specify rtol or atol")
+    elif atol is not None:
+        if atol > 0.0:
+            tolerance = atol
+    elif rtol is not None:
+        if rtol > 0.0:
+            tolerance = rtol
+            _rel = True
+
```

Comment:
Please error check that things aren't specified as normal arguments as well as kwargs. The biggies are `Workspace1`, `Workspace2`, `Tolerance`, and `ToleranceRelErr`.